templates/store-header-times.php

```diff
@@ -0,0 +1,51 @@
+<div id="vendor-store-times">
+    <div class="store-times-heading">
+        <i class="fas fa-calendar-day"></i>
+        <h4><?php echo esc_html( $times_heading ); ?></h4>
+    </div>
+    <?php
+    foreach ( $dokan_days as $day_key => $day ) :
+        $store_info   = ! empty( $dokan_store_times[ $day_key ] ) ? $dokan_store_times[ $day_key ] : [];
+        $store_status = ! empty( $store_info['status'] ) ? $store_info['status'] : 'close';
+        ?>
+        <div class="store-time-tags">
+            <div class="store-days <?php echo $today === $day_key ? 'current_day' : ''; ?>"><?php echo esc_html( $day ); ?></div>
+            <div class="store-times">
+                <?php if ( $store_status === 'close' ) : ?>
+                    <span class="store-close"><?php echo esc_html( $closed_status ); ?></span>
+                <?php endif; ?>
+
+                <?php
+                // Get store times.
+                $opening_times = ! empty( $store_info['opening_time'] ) ? $store_info['opening_time'] : [];
+
+                // If dokan pro doesn't exists then get single item.
+                if ( ! dokan()->is_pro_exists() ) {
+                    $opening_times = ! empty( $opening_times ) && is_array( $opening_times ) ? $opening_times[0] : [];
```

Comment:
As it seems, the `$opening_times` here is holding data like `[ 'time1', 'time2',.... ]`. What you intend to do is taking only one time from here but ultimately process further considering it as an array. In that case, it's better to make an array with the first element here. for example,
 `$opening_times = ! empty( $opening_times ) && is_array( $opening_times ) ? [ $opening_times[0] ] : [];`

Reply:
Even though you are casting it in array later, it's safer to do it this way here.